packages/vetra/connect.tsx

```diff
@@ -1,4 +1,3 @@
-import "@powerhousedao/connect/styles.css";
```

Comment:
Does it need to be added to the `index.html`?

Reply:
it is already added in the index.html, I'm using the same pattern from the boilerplate with the connect.css file